frontend/packages/operator-lifecycle-manager/src/components/clusterserviceversion.tsx

```diff
@@ -686,7 +864,25 @@ export const ClusterServiceVersionDetails: React.SFC<ClusterServiceVersionDetail
         <div className="co-m-pane__body-group">
           <div className="row">
             <div className="col-sm-6">
-              <ResourceSummary resource={props.obj} />
+              <ResourceSummary
+                resource={props.obj}
+                customResource={{
+                  label: 'Managed Namespaces',
+                  description: 'Operands in this Namespace are managed by the Operator.',
+                  path: 'metadata.namespace',
+                  children:
+                    olmOperatorGroup === 'global-operators' ? (
```

Comment:
Again, this logic isn't completely correct. Check if there is no `olm.targetNamespaces` annotation.

Reply:
Is there a reason we don't handle the case where there are multiple namespaces here like in the list view?

Reply:
This was @tlwu2013's rationale:

> Details view no need to display multi-NS rare case, reason being:
this is reserved for list view “all-projects” case because only admin users can view all-projects  so they have access to any projects in a cluster
details view is single NS, users may be non-admin role who only have access to their own projects
       --> therefore, the exact same logic as list view “single project” case  (see login pinned in this channel)
       --> i.e. for global Operators, show “All Namespaces” in Managed Namespaces field
                   for non-global Operators, show “[current viewing namespace]” in Managed Namespaces field
hope this helps